action/envelope.go

```diff
@@ -307,6 +307,11 @@ func (elp *envelope) loadProtoTxCommon(pbAct *iotextypes.ActionCore) error {
 		if err = tx.fromProto(pbAct); err == nil {
 			elp.common = tx
 		}
+	case BlobTxType:
+		tx := BlobTx{}
```

Comment:
should use pointer of `BlobTx`, instead of the value, which may not changes after `fromProto`